src/Vipps.net.AspCoreDemo/Startup.cs

```diff
@@ -21,7 +22,28 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            var vippsConfigurationOptions = new VippsConfigurationOptions
+            {
+                ClientId = configuration.GetValue<string>("CLIENT-ID")!,
+                ClientSecret = configuration.GetValue<string>("CLIENT-SECRET")!,
+                MerchantSerialNumber = configuration.GetValue<string>("MERCHANT-SERIAL-NUMBER")!,
+                SubscriptionKey = configuration.GetValue<string>("SUBSCRIPTION-KEY")!,
+                UseTestMode = true,
+                PluginName = Assembly.GetExecutingAssembly().GetName().Name,
+                PluginVersion =
+                    Assembly.GetExecutingAssembly().GetName().Version?.ToString() ?? "1.0.0"
+            };
```

Comment:
This does not build. `configuration` is missing. `Assembly` also doesn't exists, add a `using` statement to fix that